src/XrdPosix/XrdPosixDir.hh

```diff
@@ -40,6 +40,7 @@
 #endif
 
 #include <unistd.h>
+#include <sys/stat.h>
```

Comment:
Generally, if we don't make use of an object we simply forward declare it and the definition of the object (i.e. the include file) is placed  inthe cc file where it is used. That way we try to avoid implied dependencies on include files which happen from time to time.

Reply:
Done.